### DISCUSSION TITLE
Following shows old company repo updates

### DISCUSSION THREAD

**Discussion by subbramanil** [community/community#21140] [Other Feature Feedback, Questions, & Ideas] [answered]
I recently changed jobs, and I still see the activities on the repositories of the old company. This is weird as their repositories are private, and I am no longer a member or an external collaborator for their repos.

**Comment by MrSarno** [comment]
Does this consist of (potentially cached) old data, or are you experiencing access to activities since you left?

Sorry if it sounds like a stupid question; it's just good to make certain.

**Reply by subbramanil** [reply]
No, that's a valid question @MrSarno. 

So, I just tried a few things

- Logout, Delete all the website data/cache related to Github in the browser and log back in
- Open a new private/incognito window and log in to Github

In both scenarios, the feed continues to show the activities from repositories of the old company.

I tested these on Safari & Firefox browsers.

**Reply by subbramanil** [reply]
Also, the activities link themselves are dead links. i.e., when I open them, they show a 404 page(Which is what's expected on private repos).

**Reply by MrSarno** [reply]
I didn’t mean to clarify cached on your end, such as a browser-side cache. Sorry if I was unclear.

When I asked if you are observing activities since you left or only cached data, I’m referring to GitHub not retroactively removing (or granting) access to activities when leave or join organisations.

**Reply by subbramanil** [reply]
@MrSarno Makes sense.

**Comment by siminapasat** [comment, marked as answer]
👋 @subbramanil how long has it been since you left your previous company? hours? days? weeks? This will help us investigate. TY 😊

**Reply by siminapasat** [reply]
Following up: you will always see events in their Following feed that were created back when you had access, however as you noted you won't be able to access them and you won't see any new events.

**Reply by subbramanil** [reply]
@siminapasat It's been a week. But I can understand how the events created when I had access, showing up in the feed. I'd imagine it's similar to the contribution graph where it shows the contributions(commits) in the private repositories, although I may not be able to see the code.

If I can't see the new events, that's good. Thanks for the clarification 👍🏽  @siminapasat
